modules/core/petra/petra-sql-dsl-spi/src/test/java/com/liferay/petra/sql/dsl/spi/SQLDSLTest.java

```diff
@@ -856,14 +863,20 @@ public void testOrderBy() {
 
 		Assert.assertEquals(
 			MainExampleTable.INSTANCE.nameColumn,
-			orderByExpression.getExpression());
-		Assert.assertTrue(
-			orderByExpression.toString(), orderByExpression.isAscending());
+			orderByNameColumnExpression.getExpression());
+		Assert.assertEquals(
+			MainExampleTable.INSTANCE.descriptionColumn,
+			orderByDescriptionColumnExpression.getExpression());
+		Assert.assertFalse(
+			orderByNameColumnExpression.toString(),
+			orderByNameColumnExpression.isAscending());
+		Assert.assertFalse(
+			orderByDescriptionColumnExpression.toString(),
+			orderByDescriptionColumnExpression.isAscending());
 
 		Assert.assertEquals(
 			StringBundler.concat(
-				"select MainExample.name from MainExample order by ",
-				"MainExample.name asc, ",
+				expectedJoinStepSql, " order by MainExample.name desc, ",
```

Comment:
@yuhai Before the expected value is "MainExample.name asc", why you have to change it to "MainExample.name desc"?

Reply:
@tinatian，since I changed orderByNameColumnExpression from MainExampleTable.INSTANCE.nameColumn.ascending() to MainExampleTable.INSTANCE.nameColumn.descending() . Please see https://github.com/liferay-core-infra/liferay-portal/blob/3adc175d6fe7c8e0c38bbd85711e1923ad6b0a8a/modules/core/petra/petra-sql-dsl-spi/src/test/java/com/liferay/petra/sql/dsl/spi/SQLDSLTest.java#L850-L854.

I have two reasons to change the place.
1. MainExample.name asc assert had been done in https://github.com/liferay-core-infra/liferay-portal/blob/3adc175d6fe7c8e0c38bbd85711e1923ad6b0a8a/modules/core/petra/petra-sql-dsl-spi/src/test/java/com/liferay/petra/sql/dsl/spi/SQLDSLTest.java#L831-L835

2. The commit added MainExampleTable.INSTANCE.descriptionColumn assert, I wanted to keep same order when using order by the two columns.